compiler/dyno/include/chpl/uast/Reduce.h

```diff
@@ -73,6 +93,14 @@ class Reduce final : public Call {
     return op_;
   }
 
+  /**
+   Returns the input type, which may be empty.
+   An input type is specified as int in `(minmax(int) reduce sum)`
+*/
+  UniqueString inputType() const {
```

Comment:
I feel like this should return `Identifier*` instead. That way we have location information.

Reply:
Changed to store and return an `AstNode*`, which should always be a `FnCall`, and in the good case, will have one actual which is the `InputType`.